README.md

```diff
@@ -51,7 +51,7 @@ Environnement
 -------------
 
 The ZIM library builds using [Meson](http://mesonbuild.com/) version
-0.34 or higher. Meson relies itself on Ninja, pkg-config and few other
+0.39 or higher. Meson relies itself on Ninja, pkg-config and few other
```

Comment:
The `0.39` mesion version is only necessary to build on `win32_static` platform. (using https://github.com/kiwix/kiwix-build). For normal (native) compilation, `0.34` is enough.